documentation/design-docs/dotnet-tools.md

```diff
@@ -396,6 +396,81 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+### dotnet-stack
+
+SYNOPSIS
+
+    dotnet-stack [options] [command] [<args>]
+
+OPTIONS
+
+    --version
+        Display the version of the dotnet-trace utility.
+
+    -h, --help
+        Show command line help
+
+COMMANDS
+
+    report         Displays stack traces for the target process
+
+REPORT
+
+    dotnet-stack report -p|--process-id <pid>
+                        -n|--name <process-name>
+                        [-h|--help]
+
+    Prints the managed stack from every thread in the target process
```

Comment:
(Long term) You may want to consider adding list thread ids and stack by specified thread id.